src/config.js

```diff
@@ -29,6 +29,17 @@ module.exports = {
   vipps_merchant_serial_number: process.env.VIPPS_MERCHANT_SERIAL_NUMBER,
   vipps_api_url: process.env.NODE_ENV === "production" ? "api.vipps.no" : "apitest.vipps.no",
 
+  swish_cert: process.env.SWISH_CERT,
+  swish_cert_key: process.env.SWISH_CERT_KEY,
+  swish_payee_alias: process.env.SWISH_PAYEE_ALIAS,
+  swish_url: "https://mss.cpc.getswish.net/swish-cpcapi/",
+  swish_whitelist: [
```

Comment:
Perhaps nice to also have a link to where these 4 are documented

Reply:
Added! I had just copied the values from the Swedish repo. There were some I couldn't find documentation for however, so I removed these for now.